sdk/src/wallet/bindings/java/examples/src/Env.java

```diff
@@ -2,8 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 
 public class Env {
-    public static final String NODE = "https://api.testnet.shimmer.network";
-    public static final String EXPLORER = "explorer.shimmer.network/testnet";
+    private static string NETWORK = "testnet";
+    public static final String NODE = "https://api." + NETWORK + ".shimmer.network";
```

Comment:
Just do
```suggestion
    public static final String NODE = "https://api.testnet.shimmer.network";
```
For shimmer it's `https://api.shimmer.network` and not `staging` or `mainnet`, so this network separation doesn't really make sense

Reply:
ah thats too bad :(